abci/types/types.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package tendermint.abci.types;
```

Comment:
why remove the prefix? the prefix was added intentionally to avoid names collusion.

Reply:
```
abci/types/types.proto:2:1:Files with package "tendermint.abci.types" must be within a directory "tendermint/abci/types" relative to root but were in directory "abci/types".
```
this is the linting error that arises, i will disable it